package.json

```diff
@@ -28,16 +28,30 @@
     "url": "https://github.com/turfjs/turf-www/issues"
   },
   "homepage": "https://github.com/turfjs/turf-www",
-  "dependencies": {
-    "@turf/turf": "^3.7.4",
-    "documentation": "^4.0.0"
-  },
+  "dependencies": {},
   "devDependencies": {
     "ava": "^0.17.0",
+    "chalk": "^2.1.0",
     "copy-dir": "^0.3.0",
+    "copy-webpack-plugin": "^4.1.1",
+    "documentation": "^4.0.0",
+    "eventsource-polyfill": "^0.9.6",
+    "extract-text-webpack-plugin": "^3.0.1",
+    "friendly-errors-webpack-plugin": "^1.6.1",
+    "html-webpack-plugin": "^2.30.1",
+    "http-server": "^0.10.0",
     "load-json-file": "^3.0.0",
+    "optimize-css-assets-webpack-plugin": "^3.2.0",
+    "ora": "^1.3.0",
+    "rimraf": "^2.6.2",
+    "semver": "^5.4.1",
     "serve": "^6.2.0",
+    "shelljs": "^0.7.8",
+    "standard": "^10.0.3",
     "static-server": "^2.0.3",
+    "webpack": "^3.7.1",
```

Comment:
@rowanwins Ops, didn't notice the 2nd `package.json` in `turf-template`, we shouldn't have two `packages.json` files